dspace-api/src/main/java/org/dspace/submit/step/CompleteStep.java

```diff
@@ -87,7 +87,8 @@ public int doProcessing(Context context, HttpServletRequest request,
         boolean success = false;
         try
         {
-            if(ConfigurationManager.getProperty("workflow","workflow.framework").equals("xmlworkflow")){
+            if(StringUtils.equals(configurationService.getProperty("workflow.framework"), "xmlworkflow"))
+            {
```

Comment:
As mentioned in IRC, this change will conflict with the one in #1280 .  I think the fix in #1280 is the correct one (just for this class).